main.py

```
@@ -139,29 +174,39 @@ async def register_admin(sid, data):
 @sio.event
 async def admin_join_session(sid, data):
     """Admin joins a user's handoff session"""
     user_id = data.get("user_id")
-    admin_id = data.get("admin_id", "Admin")
+    
+    # Get admin info from socket session
+    try:
+        socket_session = await sio.get_session(sid)
+        admin_id = socket_session.get("admin_id", "Admin")
+        admin_name = socket_session.get("admin_name", "Support Agent")
+    except:
```

Ruff (0.15.4)

[error] Do not use bare `except`

(E722)